ale/isd_generate.py

```diff
@@ -51,6 +51,19 @@ def main():
              "and the default strategy of replacing their final suffix with "
              ".json will be used to generate the output file paths."
     )
+    parser.add_argument(
+        "-r", "--radius",
+        type=float,
+        default=None,
+        help="Optional spherical radius (km) override.  If not specified, the default "
+             "radius values, as defined in the NAIF kernels, will be used.  "
+             "There often cases were the defined NAIF radii are set as a triaxial "
+             "when the IAU WGCCRE report recommends a best-fit sphere for a "
+             "derived map product. For current IAU spherical recommendations see: "
+             "https://doi.org/10.1007/s10569-017-9805-5 or "
+             "http://voparis-vespa-crs.obspm.fr:8080/web/. "
+             "Make sure radius value is in kilometers."
+    )
```

Comment:
What about nargs="+" (must pass one or more values, e.g., 3396190, 3376250 OR just 3396190) or nargs="2" (must pass two values, e.g.,3396190 3396190) here and making the radius fully overridable? When the args come, as a list, you would need some logic to determine the length and apply to semi major / semi minor axes properly.

I know that the current addition fits the Mars use case, but what if I want to use old (or new) IAU radii. It seems like it would be quite nice to be able to pass these fully dynamically.

Reply:
If we want to add the ability to pass multiple radii, then we should defer to the 1 radii case as it's the most common. I do not like forcing users to enter the same radii twice when we can just add some extra logic.

Reply:
allow user to pass --semiminor as sphere or --semimajor --semiminor for ellipse.  Fighting way too with argparse to send send "only" two values (as floats) so I went with two parameters. Sort of annoying but works.